### ISSUE TITLE
Forms in a tab causes page refresh on form elements focus or key down

### ISSUE THREAD

**Issue by pilerou** [driftyco/ionic#2415]
In an app, I have tabs declared by ion-tabs and ion-tab directives.
On each tab, I have an ion-nav-view.
In my app.js file, i declared states in which i fill the ion-nav-view with a template in which I have a form.

Everything is OK when I test on Chrome.
But when I test with mobile test functionalities in Chrome (simulate device and user agent), I have a problem when I write text in a field (actually a TextArea).
The problem occurs when I emulate IOS or Android.

On Chrome emulation, the problem consists on a page refresh or it seems to flicker... Each time, the user loses the focus. After many word written, my form can even disappear from the screen.

I tested it on my iPhone 4S Safari, and the problem is quite different : 
When I focus on a field, it disappears while the keyboard stays and i can type characters which aren't written in the field.

I tried to change ion-tabs directives to css equivalent (which causes inconsistent navigation and back buttons mecanism error...).

This time, no error occurs on iPhone neither on Chrome development tools with mobile user agent.

In conclusion, the problem can be reproduced by adding a form in a view inside a ion-tabs.

I can add explanation if needed.

**Comment by Ionitron** [comment]
Greetings @pilerou!

I've closed this issue because my sensors indicated it was old and inactive, and may have already been fixed in recent versions of Ionic. However, if you are still experiencing this issue, please feel free to reopen it and include any examples and other necessary information, so that we can look into it further.

Thank you for allowing me to assist you.
